iOS Example Frame/iOS Example Frame/MainViewController.swift

```diff
@@ -119,7 +119,7 @@ final class MainViewController: UIViewController, CardViewControllerDelegate, Th
         cardViewController = createCardViewController(address: address,
                                                       phone: phone,
                                                       checkoutAPIService: checkoutAPIService)
-        cardViewController?.availableSchemes = [.visa, .mastercard, .maestro]
+        cardViewController?.availableSchemes = [.visa, .mastercard, .maestro()]
```

Comment:
Why is the being passed as maestro() ?

Reply:
This is the old screen that will get removed. New one uses the `CardScheme` interface I've had to add, whereas here it just uses Card.Scheme and is therefore dependant on passing associated value. Its using a default one (0), but syntax is still impacted